Document Update (#27786)
grafana/grafana: docs/sources/features/datasources/mssql.md

ORDER BY 1
```

> When above query is used in a graph panel the result will be two series named `Metric A` and `Metric B` with value of `valueOne` and `valueTwo` plotted over `time`.

Reviewer: ```suggestion
When the above query is used in a graph panel, the result is two series named `Metric A` and `Metric B` with value of `valueOne` and `valueTwo` plotted over `time`.
```

Author: I have made the changes as per the suggestion

Author: I have made the changes as per the @achatterjee-grafana 's suggestion

> When above query is used in a graph panel the result will be two series named `Metric A` and `Metric B` with value of `valueOne` and `valueTwo` plotted over `time`.

Reviewer 2: Change sentence to:
When the above query is used in a graph panel, it will produce two series named `Metric A` and `Metric B` with the values `valueOne` and `valueTwo` plotted over `time`.

Thanks!

Author: I have made the changes as per the suggestion.

ORDER BY 1
```

> When above query are used in a graph panel the result will be two series named `valueOne` and `valueTwo` with value of `valueOne` and `valueTwo` plotted over `time`.

Reviewer 2: Same change as above.

Author: I have made the changes as per the suggestion

> When above query is used in a graph panel the result will be two series named `valueOne` and `valueTwo` with value of `valueOne` and `valueTwo` plotted over `time`.

Reviewer: ```suggestion
When above query is used in a graph panel, the result is two series named `valueOne` and `valueTwo` with value of `valueOne` and `valueTwo` plotted over `time`.
```

Author: I have made the changes as per the @achatterjee-grafana 's suggestion

ORDER BY 1
```

> When above query is used in a graph panel the result will be two series named `Metric A` and `Metric B` with an average of `valueOne` plotted over `time`.

Reviewer: ```suggestion
When above query is used in a graph panel, the result is two series named `Metric A` and `Metric B` with an average of `valueOne` plotted over `time`.
```

Author: I have made the changes as per the @achatterjee-grafana 's suggestion

> When above query is used in a graph panel the result will be two series named `Metric A` and `Metric B` with an average of `valueOne` plotted over `time`.

Reviewer 2: Same change as above.

Author: I have made the changes as per the suggestion

> Any two series lacking a value in a 3 minute window will render a line between those two lines. You'll notice that the graph to the right never goes down to zero.

Reviewer: ```suggestion
Any two series lacking a value in a three-minute window will render a line between those two lines. You'll notice that the graph to the right never goes down to zero.
```

### Examples

{{< docs-imagebox img="/img/docs/v51/mssql_metrics_graph.png" class="docs-image--no-shadow docs-image--right" >}}

> For the following examples the database table is defined in [Time series queries](#time-series-queries). Let's say that we want to visualize 4 series in a graph panel, i.e. all combinations of columns `valueOne`, `valueTwo` and `measurement`. Graph panel to the right visualizes what we want to achieve. To solve this, we actually need to use two queries:

Reviewer: ```suggestion
For the following examples, the database table is defined in [Time series queries](#time-series-queries). Let's say that we want to visualize four series in a graph panel, such as all combinations of columns `valueOne`, `valueTwo` and `measurement`. Graph panel to the right visualizes what we want to achieve. To solve this, we need to use two queries:
```

Author: I have made the changes as per the suggestion
